js/sign/src/integrityblock.ts

```diff
@@ -0,0 +1,30 @@
+import crypto, { KeyObject } from 'crypto';
+
+// A helper function which can be used to parse string formatted keys to
+// KeyObjects.
+export function parseStringKey(unparsedKey: string): KeyObject {
+  return crypto.createPrivateKey({
+    key: unparsedKey,
+  });
+}
+
+export class IntegrityBlockSigner {
+  private key: KeyObject;
+  private webBundle: Uint8Array = new Uint8Array();
```

Comment:
```suggestion
  private webBundle: Uint8Array;
```
because it's initialized in the constructor.

Alternatively, you can declare these fields in the constructor signature like this:
```typescript
  constructor(private webBundle: Uint8Array, private key: KeyObject) {
```